Added ate_relation.py and likes_relation.py

isaiahar opened the pull request:
The ate_relation handles the '/api/v1/food_eaten' endpoint with "GET" and "POST" responses
The likes_relation handles the '/api/v1/food_liked' endpoint with "GET", "POST", and "DELETE" responses

isaiahar committed:
Added ate_relation.py and likes_relation.py

ate_relation handles the '/api/v1/food_eaten' endpoint with "GET" and "POST" responses
likes_relation handles the '/api/v1/food_liked' endpoint with "GET", "POST", and "DELETE" responses

anonymousanteater reviewed (changes requested):
Please update this to add these functions in app.py and add the db functions in db.py .

isaiahar committed:
Update db.py to handle ATE and LIKES relationships

ATE relationship: GET a list of foods that a user has eaten, POST a new food that a user has eaten.
LIKES relationship: GET a list of foods that a user has liked, POST a new food that a user has liked, DELETE an existing LIKES relationship between a user and a food

isaiahar committed:
App.py now handles ATE & LIKES endpoints

api/v1/food_eaten
api/v1/food_liked

isaiahar committed:
Fixed some 'return' statements in db.py

isaiahar committed:
Updated db.py to have correct db_driver syntax

Commit:
Merge branch 'master' into isaiahar-ate&likes-endpoint
